### PR TITLE
Calculator Design

### PR DESCRIPTION
**Pull request by maury18**
# This pull request is to check if the design of the calculator was added correctly as the requirements say
- Added components´ folder
- Added calculator.js file
- Imported calculator file into the App.js file
- Checked for linter errors

#Added required changes
Added class-based to all components

image

### CODE REVIEW

File: src/components/calculator.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
+const calculator = () => (
+  <div className="App container">
+    <div className="app">
```
Comment (KDlamini) [inline src/components/calculator.js]: The `Calculator.js` component should be class-based as per project requirements.

image

- Kindly refactor your component to use classes.
For example:
```
import React, { Component } from 'react';

class Calculator extends Component {
  render() {
      ....
    };
  }
}

export default Calculator;
```

File: src/App.js [hunk 2]
```diff
@@ -1,28 +1,8 @@
-import logo from './logo.svg';
-import './App.css';
+import calculator from './components/calculator';
 
 function App() {
   return (
```
Comment (KDlamini) [inline src/App.js]: Same here, The `App.js` component should also be class-based. Kindly refactor your component to use classes as per project requirements.

**Review by KDlamini — CHANGES_REQUESTED** [review]
### Hi @maury18 🙋‍♀️,  

Great job so far! 💯 
There are some issues that you still need to work on to improve your project according to the requirements. You are almost there! 💪🏾 

## Highlights 🌞 
✔️  Good PR description
✔️  Correct use of Gitflow 
✔️  Commit messages are meaningful
✔️  No linter errors

## Required Changes ♻️


**Kindly look through the comments below 👇 for required changes**

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

N/A

Cheers 🥂  and Happy coding!!! 👯 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me @KDlamini  in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by julie-ify — APPROVED** [review]
Hi @maury18, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Status: Approved ✔️ 

Cheers and Happy coding!👏👏👏

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**